content/integrations/faq/open-file-descriptors-metric-collection.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Process open file descriptors metric collection
+kind: faq
+further_reading:
+- link: "/integrations/process"
+  tag: "Documentation"
+  text: Learn more about the Datadog Process integrations
+---
+
+For the [Datadog - process integration](/integrations/process/) `system.processes.open_file_descriptors` metric is only available for processes running as the same user as the agent (dd-agent).
+
+If you wish to collect this metric, a workaround may be to run a small Python script as root that uses psutil to read this metric and sends it to the Datadog Agent using [DogStatsD](/developers/dogstatsd).
```

Comment:
I would maybe link to psutil, and put backticks around it: [`psutil`](https://github.com/giampaolo/psutil)